server/client.go

```diff
@@ -760,6 +760,10 @@ func (c *client) writeLoop() {
 	// Used to check that we did flush from last wake up.
 	waitOk := true
 
+	// Used to limit the wait for a signal
+	t := time.NewTimer(time.Second)
```

Comment:
Why do we create at 1 second but reset to 3?

Reply:
Because I initially decided to use 1sec but then moved to 3sec.. will change but this does not really matter because we call reset before entering the select.

Reply:
Agree was just curious.